addons/cppcheckdata.py

```diff
@@ -266,6 +304,7 @@ class Token:
     isString = False
     strlen = None
     isChar = False
+    isBoolean = False
     isOp = False
```

Comment:
should isCast,isComplex,isRestrict,isAttributeExport be added here also maybe I missed something

Reply:
It can be done, but is not really needed.

Reply:
I don't have an opinion. But if you don't add isCast here, then isCast must always be assigned in the `__init__`.

Reply:
It does make sense to make sure attributes are always initialized. Else we need to write things like this, to avoid issues.

if hasattr(scope.function, "type") and scope.function.type == 'Constructor':

I will prepare an update.